sway/desktop/output.c

```diff
@@ -808,10 +808,15 @@ static void render_output(struct sway_output *output, struct timespec *when,
 			wlr_renderer_clear(renderer, clear_color);
 		}
 
+		float opacity = workspace->sway_workspace->fullscreen->swayc->alpha;
+		workspace->sway_workspace->fullscreen->swayc->alpha = 1.0f;
```

Comment:
Not a fan of this

Reply:
Would a `float saved_alpha` field in `struct sway_view` and store/restore it when changing the fullscreen status be fine?

Reply:
Yeah, just like saved_width/saved_height, that would be better.

Reply:
Fixed